grpc-gateway/pb/cancelCommands.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package ocf.cloud.grpcgateway.pb;
+
+import "github.com/plgd-dev/cloud/resource-aggregate/pb/commands.proto";
+
+option go_package = "github.com/plgd-dev/cloud/grpc-gateway/pb;pb";
+
+message CancelResourceCommandsRequest {
+    ocf.cloud.resourceaggregate.pb.ResourceId resource_id = 1;
```

Comment:
I think we wanted to have unique correlation ids per device, so I can cancel all pending commands of one device.

Reply:
Not possible, with current DB model -> it means that we need to change DB model from resources to devices. Or we need to have stored used correlelationIDs in DB and sync them with resources.

Reply:
It is possible if somebody is interested in deleting all pending commands of one device. He could just send device id. Nope?